backend/typescript/migrations/2024.06.12T01.36.38.create-role-table.ts

```diff
@@ -1,3 +1,6 @@
+// DEPRECATED
```

Comment:
nit: we can use TODO so that vscode extensions like: https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

can detect these and make it easier for us to find and do in the future

Reply:
done